### PR TITLE
Implement comment sorting.  Fix #54

### LINKED ISSUE #54: Implement Comment Sorting
By time / upvotes / answered.

### PR DESCRIPTION
**Pull request by adambatson**
Only sorts top level comments for now.  I will work on sorting child comments if time permits.

### CODE REVIEW

File: src/main/java/com/sysc4806/PostController.java [hunk 1]
```diff
@@ -145,7 +147,29 @@ public String viewAMA(@RequestParam(value="id", required = false) Long ama, @Req
                 model.addAttribute("ama", p);
                 model.addAttribute("title", p.getTitle());
                 model.addAttribute("page", "view");
-                model.addAttribute("comments", commentRepo.findByPostAndParent(p, null));
+                if(sortby != null) { //Sort the comments
+
+                    if (sortby.equals("upvotes")) {
```
Comment (rscarson) [inline src/main/java/com/sysc4806/PostController.java]: would a switch not be cleaner here?

File: src/main/resources/templates/ama/view.html [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
-<head></head>
+<head>
```
Comment (rscarson) [inline src/main/resources/templates/ama/view.html]: Why is the javascript included here? The head in these fragments gets thrown away anyhow

File: src/main/java/com/sysc4806/PostController.java (hunk 1 above)
Comment (adambatson) [inline src/main/java/com/sysc4806/PostController.java]: fine

File: src/main/resources/templates/ama/view.html (hunk 2 above)
Comment (adambatson) [inline src/main/resources/templates/ama/view.html]: left over from debugging my bad

**Pull request merged**